tensorflow/lite/micro/tools/make/targets/ceva_makefile.inc

```diff
@@ -0,0 +1,110 @@
+
+ifeq ($(TARGET), ceva)
+
+TARGET_ARCH :=
+
+ifeq ($(TARGET_ARCH), )
+  $(error TARGET_ARCH must be specified on the command line)
+
+endif# Create a cflag based on the specified TARGET_ARCH. For example:
+#   TARGET_ARCH=hifimini --> -DHIFIMINI
+#   TARGET_ARCH=fusion_f1 --> -DFUSION_F1
+TARGET_ARCH_DEFINES := -D$(shell echo $(TARGET_ARCH) | tr [a-z] [A-Z])
```

Comment:
update the comment based on your values for TARGET_ARCH? Also, it looks like you are already selecting target_arch to be uppercase with underscores so this may not be needed.